python/BUILDING.md

```diff
@@ -3,20 +3,37 @@
 This document describes how to build and install Ice for Python from source.
 You can also download and install a [binary distribution].
 
-* [Building with Pip](#building-with-pip)
+* [Building with Build Python Package Builder](#building-with-build-python-package-builder)
 * [Building with Visual Studio 2022 and MSBuild (Python 3\.12 for Windows)](#building-with-visual-studio-2022-and-msbuild-python-312-for-windows)
 * [Building on Linux or macOS](#building-on-linux-or-macos)
 * [Configuring your Environment for Python](#configuring-your-environment-for-python)
 * [Running the Python Tests](#running-the-python-tests)
 
-## Building with Pip
+## Building with Build Python Package Builder
 
-You can build the Ice for Python extension from source using `pip`:
+You can build the Ice for Python extension using `build` Python package builder. Follow these steps:
+
+Ensure you have pip installed
+
+```shell
+python -m ensurepip --upgrade
+```
+
+Install the build module
 
 ```shell
-pip install <URL of Ice source distribution for Python>
+pip install build
```

Comment:
Should we update this to use a virtual environment and avoid issues with pip global install?

Reply:
I think this is a detail for users to worry about depending on their setup. There are lots of popular ways to use python like `pyenv` where using a virtual env is not required. 

If you're doing serious Python development it's very unlikely you're using the global Python.

Reply:
Probably.